src/activities/assignments/AssignmentEntity.js

```diff
@@ -258,4 +258,28 @@ export class AssignmentEntity extends Entity {
 		];
 		await performSirenAction(this._token, action, fields);
 	}
+
+	annotationToolsAvailable() {
+		return this._entity.getSubEntityByClasses(['annotations', 'enabled']);
```

Comment:
I'm not 100% sure, but I _suspect_ you actually want `getSubEntitiesByClasses` (plural) here? `getSubEntityByClasses` (singular) will just return the first entity that matches the requirements, whereas `getSubEntitiesByClasses` (plural) will return an array of all entities that match the requirements.

Reply:
I just want the single `annotations` sub-entity that contains the action and the current state of annotations for the assignment. There should be only one sub-entity with an `annotations` class per assignment.

Reply:
Shouldn't we ideally look for things by relationship and not their class?

Reply:
Actually, that's a better point. `getSubEntityByClass` can generally be replaced with a call to `getSubEntityByRel`.